README.md

```diff
@@ -3,8 +3,33 @@
 This application is used for exporting Yolo V5, V6 and V7 object detection models for OAKs.
 
 ## Running the app locally
-To run the application locally you need to do 2 additional things.
-### Step 1: Updating the `nginx.conf` file
+To run the application locally you need to do these folllowing steps.
+### Step 1: Cloning all repositories
+```
+# Clone tools repository
```

Comment:
Can't these be cloned with sumodules init and recursive pull? @HonzaCuhel

Reply:
Thanks for the suggestion, will look into it!

Reply:
@HonzaCuhel Thanks, can we store the commits of the repo directly instead of having to manually reset them?

Reply:
@tersekmatija I am not aware of such possibility, I searched and asked ChatGPT

Reply:
Updated!